lib/mayaUsd/render/vp2RenderDelegate/mayaPrimCommon.cpp

```diff
@@ -702,72 +701,89 @@ void MayaUsdRPrim::_UpdatePrimvarSourcesGeneric(
 }
 
 #ifdef MAYA_HAS_DISPLAY_LAYER_API
-void MayaUsdRPrim::_PopulateDisplayLayerModes(
-    const MString&     pathString,
+void MayaUsdRPrim::_ProcessDisplayLayerModes(
```

Comment:
The function actually fills the DisplayLayerModes, so a better name might be _FillDisplayLayerModes or _InitializeDisplayLayerModes. Again, not a deal breaker, really nit picking.

Reply:
Maybe it could even be a member function on DisplayLayerModes, along with the clearing you do below: fill() and clear()?